Tests/PHPUnit/Utils/Fakes/View/FakeView.php

```diff
@@ -36,7 +36,7 @@ public function assign($spec, $value = null, $nocache = null, $scope = null)
      */
     public function getAssign($spec = null)
     {
-        if (!$spec === null) {
+        if ($spec === null) {
```

Comment:
sure?

Reply:
Yeah I am. Why would you want the full array if you specify a key? It's how the view in Shopware works as well. You get everything if you don't provide a key, but just one item if you do provide a key.

Reply:
i was just asking because u changed the condition (accidentally) :D

Reply:
Not accidentally ;-)